### PR TITLE
Fix doc and makefile issue for hawq docker

### PR DESCRIPTION
**Pull request by ginobiliwang**
make pull command will fail without remote docker registry, so add local docker registry operations.

### CODE REVIEW

File: contrib/hawq-docker/Makefile [hunk 1]
```diff
@@ -203,8 +203,14 @@ remove-data:
 
 pull:
 	@echo latest images
-	docker pull hawq/hawq-dev:$(OS_VERSION)
-	docker pull hawq/hawq-test:$(OS_VERSION)
+	#For remote registry
+	#docker pull hawq/hawq-dev:$(OS_VERSION)
```
[the remark this answers is not in the document]
Comment (ginobiliwang) [reply contrib/hawq-docker/Makefile]: ok.

File: contrib/hawq-docker/Makefile [hunk 2]
```diff
@@ -203,8 +203,14 @@ remove-data:
...
+	#docker pull hawq/hawq-test:$(OS_VERSION)
+	#For local registry
```
[the remark this answers is not in the document]
Comment (ginobiliwang) [reply contrib/hawq-docker/Makefile]: Ok. Comment is primarily in README.md, will also add some comments in Makefile.

File: contrib/hawq-docker/README.md [hunk 3]
```diff
@@ -83,6 +81,38 @@ Type "help" for help.
 
 postgres=# 
 ```
+# Store docker images in local docker registry
+
+After your hawq environment is up and running, you could draft a local docker registry to store your hawq images locally for further usage.
+* pull and run a docker registry
+```
+docker pull registry
+docker run -d -p 127.0.0.1:5000:5000 registry
```
Comment (radarwave) [inline contrib/hawq-docker/README.md]: Another 127.0.0.1, I guess it can be change to localhost too?

Comment (ginobiliwang) [reply contrib/hawq-docker/README.md]: I could not use "localhost"  here, i have tried to use no ip address instead.
FYI.

```
root@localhost incubator-hawq]# docker run -d -p localhost:5000:5000 registry
/usr/bin/docker-current: Invalid ip address: localhost.
See '/usr/bin/docker-current run --help'.
[root@localhost incubator-hawq]# docker run -d -p 5000:5000 registry
40758a8d59f51b6ae345595f71e7787e2245cdbe48cad61c024196c94fbdab4c
[root@localhost incubator-hawq]# docker ps
CONTAINER ID        IMAGE                    COMMAND                  CREATED             STATUS              PORTS                    NAMES
40758a8d59f5        registry                 "/entrypoint.sh /e..."   7 seconds ago       Up 6 seconds        0.0.0.0:5000->5000/tcp   nifty_shirley
629ab67b77e8        hawq/hawq-test:centos7   "entrypoint.sh bash"     4 days ago          Up 4 days                                    centos7-datanode3
60ac39e610b7        hawq/hawq-test:centos7   "entrypoint.sh bash"     4 days ago          Up 4 days                                    centos7-datanode2
6ba58cfaf591        hawq/hawq-test:centos7   "entrypoint.sh bash"     4 days ago          Up 4 days                                    centos7-datanode1
b6e65bc29227        hawq/hawq-test:centos7   "entrypoint.sh bash"     4 days ago          Up 4 days                                    centos7-namenode
[root@localhost incubator-hawq]#
```

**Comment by ginobiliwang** [comment]
Hi @radarwave , i do not have Apache Jira created for this PR. Should i create it by myself, and should i create an apache account to draft the jira?

**Comment by ginobiliwang** [comment]
I have create one jira.

https://issues.apache.org/jira/browse/HAWQ-1655

**Pull request closed without merging**